client/src/js/app.js

```diff
@@ -326,6 +325,26 @@ function bhimaConfig($stateProvider, $urlRouterProvider, $urlMatcherFactoryProvi
     templateUrl : 'partials/purchases/create/create.html'
   })
 
+  /* cashflow page */
+  .state('cashflow', {
```

Comment:
Very nice state setup.

Following #494, it would be best to put these routes in a file `cashflow.routes.js` next to the cash flow controllers/templates.   It would be attached to the `bhima.routes` angular module.